File: frontend/src/app/domain/Symptom.ts
export class Symptom {

  public id: number;

  public name: string;
  public value: string;

  constructor();

  constructor(name: string, value: string, id: number);

  constructor(name?: string, value?: string, id?: number) {
    this.name = name;
    this.value = value;
    this.id = id;
  }
}

File: src/main/java/com/ftn/cdss/model/Symptom.java
package com.ftn.cdss.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name="symptoms")
public class Symptom {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "symptom_id")
    private Long id;

    private String name;

    private Double value;

    private boolean active = true;
}
